tools/test.py

```diff
@@ -28,6 +28,14 @@ def parse_args():
         help='directory where painted images will be saved. '
         'If specified, it will be automatically saved '
         'to the work_dir/timestamp/show_dir')
+    parser.add_argument(
```

Comment:
why do need to specify task? If we use test.py, the visualized should be correctly initialized through configs and we should set it through config.

Reply:
Thanks for your patient answer. We will merge this PR first, then consider making it simpler.